tao_compiler/mlir/disc/transforms/element_type_converter.cc

```diff
@@ -158,6 +158,42 @@ struct ConvertReduceOpWithSmallWidthIntType
   }
 };
 
+template <typename OpTy>
+struct ConvertConvOp : public OpRewritePattern<OpTy> {
+  using OpRewritePattern<OpTy>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(OpTy op,
+                                PatternRewriter& rewriter) const override {
+    Location loc = op.getLoc();
+    Value lhs = op->getOperand(0);
+    Value rhs = op->getOperand(1);
```

Comment:
just curious about why only convert first 2 operands ...

Reply:
ConvOp only has 2 operands. The third operand of DynamicConvOp is padding size.